build.sbt

```diff
@@ -338,6 +325,7 @@ val mtagsSettings = List(
 lazy val mtags3 = project
   .in(file(".mtags"))
   .settings(
+    unmanagedSourceDirectories.in(Compile) := Seq(),
```

Comment:
In order not to add `.mtags` directories, although it seems that sbt now created the `.mtags` directory anyway.